src/System.Memory/src/System.Memory.csproj

```diff
@@ -21,7 +21,13 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
   <ItemGroup>
+    <Compile Include="System\Buffers\IMemoryList.cs" />
```

Comment:
nit: move everything under System\Buffers down with the rest of the files (such as System\Buffers\ArrayMemoryPool.cs).